docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -229,24 +261,43 @@ spec:
 For details, see the [LogPipeline specification file](https://github.com/kyma-project/kyma/blob/main/components/telemetry-operator/apis/telemetry/v1alpha1/logpipeline_types.go).
 
 | Parameter | Type | Description |
-|-|-|-|
-| filters | []object | A list of [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters) to apply to the logs processed by the pipeline. Filters are executed in sequence, as defined. They are executed before logs are buffered, and with that, are not executed on retries.|
-| filters[].custom | string | The actual filter definition in the syntax of Fluent Bit.|
-| output | object | [Fluent Bit outputs](https://docs.fluentbit.io/manual/pipeline/outputs).|
-| output.custom | string | The actual output definition in the syntax of Fluent Bit.|
+|---|---|---|
+| filters | []object | List of [Fluent Bit filters](https://docs.fluentbit.io/manual/pipeline/filters) to apply to the logs processed by the pipeline. Filters are executed in sequence, as defined. They are executed before logs are buffered, and with that, are not executed on retries.|
+| filters[].custom | string | Actual filter definition in the Fluent Bit syntax.|
+| output | object | [Fluent Bit output](https://docs.fluentbit.io/manual/pipeline/outputs) where the logs should be pushed to. Only one output can be specified. |
+| output.http | object | [Fluent Bit http output](https://docs.fluentbit.io/manual/pipeline/outputs/http). |
+| output.http.compress | string | Payload compression mechanism. |
+| output.http.dedot | boolean | Indicates if *dedotting* (replacing dots with underscores) should be applied on the log field names `kubernetes.annotations` and `kubernetes.labels`. Default is `false`. |
+| output.http.format | string | Data format to be used in the HTTP request body. Default is `json`. |
+| output.http.host | object | IP address or hostname of the target HTTP server. |
+| output.http.host.value | string | Actual host value, can contain references to secret values. |
+| output.http.host.valueFrom.secretKeyRef | object | Reference to a key in a Secret. `name` and `namespace` of the Secret, as well as the name of the `key`, must be provided. |
+| output.http.password | object | Basic Auth password. |
+| output.http.password.value | string | Actual password value, can contain references to secret values. |
+| output.http.password.valueFrom.secretKeyRef | object | Reference to a key in a Secret. `name` and `namespace` of the Secret, as well as the name of the `key`, must be provided. |
+| output.http.port | string | TCP port of the target HTTP server. Default is `443`.  |
+| output.http.tls | object | TLS Configuration of the HTTP target server.  |
+| output.http.tls.disabled | boolean | Indicates if TLS is disabled or enabled. Default is `false`. |
```

Comment:
Just guessing what means what. 

```suggestion
| output.http.tls.disabled | boolean | If `true`, TLS is disabled. Default is `false`, so TLS is enabled. |
```

Why on earth is this parameter called "disabled" and not "enabled" , so `true` would mean = enabled. 
Right now, we're using double negative = positive, "disabled=false" = TLS enabled, which is confusing.

Reply:
also thought the same  😅

Reply:
Is it too late to change that?

Reply:
Reason that the name is disabled: TLS enabled should be the default, flipping the parameter to enabled, it is not possible to distinguish if the user explicitly set enabled=false or if the parameter was not set at all (and thus TLS should be active).